controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporter.java

```diff
@@ -78,15 +78,33 @@ public void maintain() {
 
     private void reportAuditLog() {
         AuditLog log = controller().auditLogger().readLog();
+        HashMap<String, HashMap<String, Integer>> metricCounts = new HashMap<>();
+
         for (AuditLog.Entry entry : log.entries()) {
             String[] resource = entry.resource().split("/");
-            String operationMetric;
-            if(resource.length > 1 && resource[1] != null) {
+            if((resource.length > 1) && (resource[1] != null)) {
                 String api = resource[1];
-                operationMetric = OPERATION_PREFIX + api;
-                Metric.Context context = metric.createContext(Map.of("operator", entry.principal()));
-
-                metric.add(operationMetric, 1, context);
+                String operationMetric = OPERATION_PREFIX + api;
+
+                if (metricCounts.containsKey(operationMetric)) {
+                    HashMap<String, Integer> dimension = metricCounts.get(operationMetric);
```

Comment:
You can simplify this and drop a key lookup by always doing `metricCounts.get` and checking for `null` value  instead of the explicit `containsKey`. Same for the code below.